src/api/vdom.js

```diff
@@ -258,7 +267,7 @@ const newElementOpenEnd = wrapIdomFunc(elementOpenEnd);
 
 // Standard open / closed overrides don't need to reproduce internal behaviour
 // because they are the ones referenced from *End and *Start.
-const newElementOpen = wrapIdomFunc(elementOpen, stackOpen);
+const newElementOpen = attributesContext(wrapIdomFunc(elementOpen, stackOpen));
```

Comment:
How can we be sure that this is the only function we need to wrap? Are there any performance implications with this do you think?